cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
   }
 
-  typeName(name) {
+  typeName() {
     this.nameField.type(name, { force: true });
```

Comment:
The typeName method is missing a 'name' parameter. You need to pass the 'name' argument to this method to correctly type into the name field.